managed/services/supervisord/supervisord.go

```diff
@@ -436,6 +438,8 @@ func (s *Service) marshalConfig(tmpl *template.Template, settings *models.Settin
 		"ClickhouseBlockSize":      clickhouseBlockSize,
 	}
 
+	s.addPostgresParams(templateParams)
```

Comment:
To be honest I'd prefer to not have this func and fill templateParams right here, same as we do for CH right above.

Reply:
same with me. Only supervisor refactoring goes beyond this task. PR is too big already.